lib/common/tests/strings/pcmk__strkey_table_test.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <setjmp.h>
+#include <cmocka.h>
+
+#include <glib.h>
+
+static void
+store_strs(void **state)
+{
+    GHashTable *tbl = NULL;
+
+    tbl = pcmk__strkey_table(free, free);
+    assert_non_null(tbl);
+
+    assert_true(g_hash_table_insert(tbl, strdup("key-abc"), strdup("val-abc")));
```

Comment:
I'm confused ... g_hash_table_insert() returns void

Reply:
"Starting from GLib 2.40, this function returns a boolean value to indicate whether the newly added value was already in the hash table or not."

Reply:
Cool, we require 2.42+